receiver/opencensusreceiver/opencensus.go

```diff
@@ -72,7 +73,12 @@ const source string = "OpenCensus"
 // as the various Stop*Reception methods to end it.
 func New(addr string, tc consumer.TraceConsumer, mc consumer.MetricsConsumer, opts ...Option) (*Receiver, error) {
 	// TODO: (@odeke-em) use options to enable address binding changes.
-	ln, err := net.Listen("tcp", addr)
+	network := "tcp"
+	if strings.HasPrefix(addr, "unix:") {
+		network = "unix"
```

Comment:
Instead of parsing the endpoint it seems better to add a `network` (or perhaps `transport`) configuration setting for the receiver with "tcp" as the default value. This way the parsing an munging with endpoint can be avoided.

Reply:
I've made this change - the opencensusreceiver now takes an additional `transport` argument that defaults to `tcp`.

Reply:
I believe it should work on Windows 10+. I can't test at the moment as the windows build doesn't work (there's a problem with the prometheus client_golang as far as I can tell).